src/tests/components/ArticleTag.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "../setup";
```

Comment:
You don't need to import the '../setup' file to your test files. It is recognized from the jest.config.js file. Nice job so far

Reply:
I don't have the jest.config.js file yet I'll have to get it from your branch. But thanks for the feedback.